pkg/cloud/openstack/clients/machineservice.go

```diff
@@ -269,18 +273,64 @@ func (is *InstanceService) InstanceCreate(name string, config *openstackconfigv1
 		}
 	}
 	userData := base64.StdEncoding.EncodeToString([]byte(cmd))
-	createOpts = servers.CreateOpts{
+	clusterTag := ""
+	for _, tag := range config.Tags {
+		if string.Contains(tag.Name, "ClusterID") {
```

Comment:
It appears you have the wrong package name here, which is causing the tests to fail as the code is unbuildable since there isn't a `string` package
```suggestion
		if strings.Contains(tag.Name, "ClusterID") {
```